Various README improvements (#531)
chatmail/server: README.md

# Chatmail servers for secure instant messaging 

Chatmail servers are interoperable email routing machines optimized for:

- **Convenience:** Low friction instant onboarding

> - **Privacy:** No name, telephone numbers, or email required to sign-up permitting you to create unique and disposable identities. Push notification architecture is privacy-preserving. (Apple, Google, and Huawei)

Reviewer: when making a bullet-point list i suggest to have each point consume similarly many characters. 

```suggestion
- **Privacy:** No name, phone numbers,  email required or collected 
- **Instant:** Privacy-preserving push notifications for Apple, Google, and Huawei
```

- **Speed:** Message delivery in well under a second.

- **Security:** Strict TLS, DKIM and OpenPGP with metadata-minimization rules enforced.

> - **Reliability:** No spam or IP reputation checks to disrupt message delivery. Default per-user message rate limit is high enough for realtime chats.

Reviewer: ```suggestion
- **Reliability:** No spam or IP reputation checks, rate-limits suitable for realtime chats.
```

- **Efficiency:** Messages are only stored for transit and removed automatically.

This repository contains everything needed to setup a ready-to-use Chatmail server
comprised of a minimal setup of the battle-tested 
[Postfix SMTP](https://www.postfix.org) and [Dovecot IMAP](https://www.dovecot.org) services.

The automated setup is designed and optimized for providing Chatmail addresses
for immediate permission-free onboarding through chat apps and bots.
Chatmail addresses are automatically created at first login,
after which the initially specified password is required
for sending and receiving messages through them.

Please see [this list of known apps and client projects](https://chatmail.at/apps.html) which offer instant onboarding on Chatmail servers

> and [this list of known public 3rd party Chatmail servers](https://delta.chat/en/chatmail).

Reviewer: ```suggestion
and [this list of known public 3rd party Chatmail servers](https://chatmail.at/server).
```
